lib/builderator/config/rash.rb

```diff
@@ -53,7 +53,7 @@ def merge!(other)
           k = k.gsub(/\-/, '_') if k.is_a?(String)
 
           ## Merge Arrays
-          next self[k] += v if fetch(k, nil).is_a?(Array) && v.is_a?(Array)
+          next self[k] |= v if fetch(k, nil).is_a?(Array) && v.is_a?(Array)
```

Comment:
The issue was that during re-compiles, elements were just being re-appended to list-attributes